### PR TITLE
Update repo caching

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Update repo caching section below so it includes info about the situations when dbt Cloud uses the cached copy instead

https://docs.getdbt.com/docs/deploy/deploy-environments#git-repository-caching 

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Needs review from product team

### CODE REVIEW

File: website/snippets/_cloud-environments-info.md [hunk 1]
```diff
@@ -42,6 +42,12 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+Below lists the situations when dbt Cloud uses the cached copy:
+
+- Git authentication fails.
+- There are syntax errors in the `packages.yml` file. To catch these errors sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  
+- A package is incompatible with the dbt version being used. To catch this incompatibility sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  
+
```
Comment (dbactual) [inline website/snippets/_cloud-environments-info.md:51]: For completeness, don't we also want to include failures of 3rd party services (github, dbt package hub, etc.) in this section?

Comment (nghi-ly) [reply website/snippets/_cloud-environments-info.md:51]: great flag! i agree with you. will update shortly

File: website/snippets/_cloud-environments-info.md [hunk 2]
```diff
@@ -42,6 +42,13 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+Below lists the situations when dbt Cloud uses the cached copy:
```
Comment (mirnawong1) [inline website/snippets/_cloud-environments-info.md]: i've read and been told that we should avoid using directional language like below or above for accessibility reasons
```suggestion
dbt Cloud will use the cached copy of your project's Git repo under these circumstances:
```

File: website/snippets/_cloud-environments-info.md [hunk 3]
```diff
@@ -42,6 +42,13 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
...
+
+- Outages from third-party services (for example, your Git provider).
+- Git authentication fails.
+- There are syntax errors in the `packages.yml` file. To catch these errors sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).
```
Comment (mirnawong1) [inline website/snippets/_cloud-environments-info.md]:
```suggestion
- There are syntax errors in the `packages.yml` file. You can set up and use [continuous integration (CI)](/docs/deploy/continuous-integration) to find these errors earlier.
```

File: website/snippets/_cloud-environments-info.md [hunk 4]
```diff
@@ -42,6 +42,13 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+Below lists the situations when dbt Cloud uses the cached copy:
+
+- Outages from third-party services (for example, your Git provider).
+- Git authentication fails.
+- There are syntax errors in the `packages.yml` file. To catch these errors sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).  
+- A package is incompatible with the dbt version being used. To catch this incompatibility sooner, set up and use [continuous integration (CI)](/docs/deploy/continuous-integration).
```
Comment (mirnawong1) [inline website/snippets/_cloud-environments-info.md]: sugg active voice
```suggestion
- If a package doesn't work with the current dbt version. You can set up and use [continuous integration (CI)](/docs/deploy/continuous-integration) to identify this issue sooner.
```

File: website/snippets/_cloud-environments-info.md [hunk 5]
```diff
@@ -42,6 +42,13 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+Below lists the situations when dbt Cloud uses the cached copy:
+
+- Outages from third-party services (for example, your Git provider).
```
Comment (mirnawong1) [inline website/snippets/_cloud-environments-info.md]: adding a non git related service to be extra explicit

```suggestion
- Outages from third-party services (for example, the [dbt package hub](https://hub.getdbt.com/)).
```

File: website/snippets/_cloud-environments-info.md (hunk 2 above)
Comment (nghi-ly) [reply website/snippets/_cloud-environments-info.md]: good flag!

File: website/snippets/_cloud-environments-info.md [hunk 6]
```diff
@@ -42,6 +42,13 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+dbt Cloud will use the cached copy of your project's Git repo under these circumstances:
+
+- Outages from third-party services (for example, the [dbt package hub](https://hub.getdbt.com/)).
+- Git authentication fails.
+- There are syntax errors in the `packages.yml` file. You can set up and use [continuous integration (CI)](/docs/deploy/continuous-integration) to find these errors earlier.
```
Comment (nghi-ly) [inline website/snippets/_cloud-environments-info.md]:
```suggestion
- There are syntax errors in the `packages.yml` file. You can set up and use [continuous integration (CI)](/docs/deploy/continuous-integration) to find these errors sooner.
```

File: website/snippets/_cloud-environments-info.md [hunk 7]
```diff
@@ -42,6 +42,13 @@ For improved reliability and performance on your job runs, you can enable dbt Cl
 
 dbt Cloud caches your project's Git repo after each successful run and retains it for 8 days if there are no repo updates. It caches all packages regardless of installation method and does not fetch code outside of the job runs. 
 
+dbt Cloud will use the cached copy of your project's Git repo under these circumstances:
+
+- Outages from third-party services (for example, the [dbt package hub](https://hub.getdbt.com/)).
```
Comment (mirnawong1) [inline website/snippets/_cloud-environments-info.md:47]: i think it is helpful to have another example, on top of your git one. so adding the git bc that's def a big example that would be helpful for users to know

```suggestion
- Outages from third-party services (for example,  your Git provider or the [dbt package hub](https://hub.getdbt.com/)).
```

**Review by mirnawong1 — APPROVED** [review]
non blocker suggestion and am approving to not block you. lgtm and repo caching is so cool!

File: website/snippets/_cloud-environments-info.md (hunk 7 above)
Comment (nghi-ly) [reply website/snippets/_cloud-environments-info.md:47]: the Git example is actually mentioned in the screenshot so we're good there

**Pull request merged**